### ISSUE TITLE
conda-forge Incorrect INTERFACE_LINK_LIBRARIES

### ISSUE THREAD

**Issue by cheng-chi** [xtensor-stack/xtensor-io#69]
When using xtensor-io with conda, the INTERFACE_LINK_LIBRARIES in xtensor-ioTargets.cmake contains incorrect library paths such as "/home/conda/feedstock_root/build_artifacts/xtensor-io_1576590031955/_build_env/x86_64-conda_cos6-linux-gnu/sysroot/usr/lib/librt.so" and is causing linker errors. I'm using the [linux64](https://anaconda.org/conda-forge/xtensor-io/0.9.0/download/linux-64/xtensor-io-0.9.0-he513fc3_0.tar.bz2) package.

**Comment by SylvainCorlay** [comment]
Hello @cheng-chi thank you for reporting. Which version are you using?

**Comment by cheng-chi** [comment]
> Hello @cheng-chi thank you for reporting. Which version are you using?

Thanks for the fast reply! I'm using 0.9.0. The problem can also be discovered by decompressing the [conda-forge tarball](https://anaconda.org/conda-forge/xtensor-io/0.9.0/download/linux-64/xtensor-io-0.9.0-he513fc3_0.tar.bz2)

**Comment by SylvainCorlay** [comment]
Indeed, everything should have `anaconda1anaconda2anaconda3` instead, which is replaced by the correct path.
